toolkit/tools/internal/sliceutils/sliceutils.go

```diff
@@ -72,7 +73,16 @@ func SetToSlice[T comparable](inputSet map[T]bool) []T {
 	return outputSlice[:index]
 }
 
-// SliceToSet converts a slice of K to a map[K]bool.
+// SetToSliceAll converts a map[K]V to a slice containing the map's keys.
+func SetToSliceAll[K comparable, V any](inputSet map[K]V) []K {
```

Comment:
You could call this function `MapKeys`.

Reply:
I think `SetToSliceAll` or `SetToSliceWithRepetition` is nice, as it makes it clearer that this is an alternative to the plain `SetToSlice`. `MapKeys` makes feels a bit unclear to me. How are we mapping the keys? According to what method?

Reply:
Yes, I have kept called it `SetToSlice` only to keep it consistent with the rest of sliceutils. We're using `set` instead of `map` throughout the codebase. I'm planning to make another PR to fix all `set(s)` to `map(s)` wherever applicable in the toolkit codebase.